nvme_gw_config.py

```diff
@@ -22,14 +22,26 @@ def __init__(self, gw_config_filename):
         self.nvme_gw_config = configparser.ConfigParser()
         self.nvme_gw_config.read(gw_config_filename)
 
-    def get(self, section, param):
-        return self.nvme_gw_config.get(section, param)
+    def get(self, section, param, value=None):
```

Comment:
I would prefer overloading these.

Reply:
Could you please provide an example of what you have in mind?

Overloading is not supported by python and we seem to need a module like multipledispatch [1], which looks cleaner but it introduces this module dependency. Not sure it is worth for this small thing.

[1] https://pypi.org/project/multipledispatch/

Reply:
Sorry there was a *not* missing (i.e. having an extra function like get_with_fallback or similar) or I'm also fine with giving the parameter a more descriptive name like `fallback` so it is clear what it does.

Reply:
Do you like `get_with_default` (or  `get_with_fallback`) name?